lib/resources/auditd.rb

```diff
@@ -0,0 +1,236 @@
+    filter.add_accessor(:where)
+          .add_accessor(:entries)
+          .add(:file,         field: 'file')
+          .add(:list,         field: 'list')
+          .add(:action,       field: 'action')
+          .add(:fields,       field: 'fields')
+          .add(:fields_nokey, field: 'fields_nokey')
+          .add(:syscall,      field: 'syscall')
+          .add(:key,          field: 'key')
+          .add(:arch,         field: 'arch')
+          .add(:path,         field: 'path')
+          .add(:permissions,  field: 'permissions')
+          .add(:exit,         field: 'exit')
+
+    filter.connect(self, :params)
+
+    def status(name = nil)
+      return @legacy.status(name) if @legacy
+
+      @status_content ||= inspec.command('/sbin/auditctl -s').stdout.chomp
+      @status_params ||= Hash[@status_content.scan(/^([^ ]+) (.*)$/)]
+
+      return @status_params[name] if name
+      @status_params
+    end
+
+    def parse_content
+      @lines = @content.lines.map(&:chomp)
+
+      lines.each do |line|
+        if is_file_syscall_syntax?(line)
+          get_file_syscall_syntax_rules(line)
+        end
+
+        if is_syscall?(line)
+          get_syscall_rules(line)
+
+        elsif is_file?(line)
+          get_file_rules(line)
+        end
+      end
+    end
+
+    def get_file_syscall_syntax_rules(line)
+      file = get_file_syscall_syntax line
+      action, list = get_action_list line
+      fields = get_fields line
+      key_field, fields_nokey = remove_key fields
+      key = get_key_from_field key_field.join('')
+      perms = get_perms fields
+
+      @params.push(
+        {
+          'file' => file,
+          'list' => list,
+          'action' => action,
+          'fields' => fields,
+          'permissions' => perms,
+          'key' => key,
+          'fields_nokey' => fields_nokey,
+        },)
+    end
+
+    def get_syscall_rules(line)
+      syscalls = get_syscalls line
+      action, list = get_action_list line
+      fields = get_fields line
+      key_field, fields_nokey = remove_key fields
+      key = get_key_from_field key_field.join('')
+      arch = get_arch fields
+      path = get_path fields
+      perms = get_perms fields
+      exit_field = get_exit fields
+
+      syscalls.each do |s|
+        @params.push(
+          {
+            'syscall' => s,
+            'list' => list,
+            'action' => action,
+            'fields' => fields,
+            'key' => key,
+            'arch' => arch,
+            'path' => path,
+            'permissions' => perms,
+            'exit' => exit_field,
+            'fields_nokey' => fields_nokey,
+          },)
+      end
+    end
+
+    def get_file_rules(line)
+      file = get_file line
+      perms = get_permissions line
+      key = get_key line
+
+      @params.push(
+        {
+          'file' => file,
+          'key' => key,
+          'permissions' => perms,
+        },)
+    end
+
+    def to_s
+      'Audit Daemon Rules'
```

Comment:
Because of our silly CLI parser, I'd recommend we keep this to two space-separate words. Otherwise, `Rules` will get pushed down to the next line. It's a long-standing bug and we'll fix it eventually :)

I'd recommend `auditd rules` for now.